transit_odp/pipelines/pipelines/dataset_etl/utils/dataframes.py

```diff
@@ -25,6 +25,34 @@
 logger = logging.getLogger(__name__)
 
 
+def create_naptan_flexible_zone_df_from_queryset(queryset):
+    flexible_zone = (
+        {
+            "naptan_id": obj.naptan_stoppoint_id,
+            "flexible_location": Point(obj.location.x, obj.location.y),
```

Comment:
Could we rename `flexile_location` to `geometry` which would make `transform_geometry` method redundant?

Reply:
we can discuss this